README.md

```diff
@@ -58,8 +58,10 @@ Coming features will helpful
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
->Click this Link to View [Live Versio](https://github.com/tchasinga/To-do-list-Microverse) coming soon
-but now you've only access to source code thankns you 
+> Coming soon , we'll be ovejoyed to see you back , thanks for your understanding
+```
+ but now the access is N/A
+``` 
```

Comment:
- [ ] @tchasinga and @Aqurds kindly consider deleting the live demo section as previously requested, this will make your readme more neat.